generators/add-action/customer-profile/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+
+class CustomerProfileGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {
+      description: 'This is a sample action showcasing how to access an external Adobe Experience Platform: Realtime Customer Profile API',
+      // eslint-disable-next-line quotes
+      requiredParams: `['tenant', 'orgId', 'apiKey']`,
+      // eslint-disable-next-line quotes
+      importCode: `const CustomerProfileSDK = require('@adobe/aio-lib-customer-profile')`,
+      responseCode: `// initialize sdk
+    const client = await CustomerProfileSDK.init(params.tenant, params.orgId, params.apiKey, token)
+    // call methods, eg getProfile
+    const { status, body } = await client.getProfile({
```

Comment:
Two things:
- This won't work as the sdk is throwing an error if the status is not valid
to keep it easy and aligned with the other templates I would just do 
```
const profile = await client.getProfile({...})
const response = {
statusCode: 200,
body: profile
}
```
- even with getProfile I am getting a similar issue on my generated app, are you getting the same (from the logs with `aio app logs` after deploy)?
```
error: [CustomerProfileAPISDK:ERROR_ENTITIES] Error 400 - Bad Request ({"message":"UPS-ACC-1000002:Some parameters are not specified - \"entityId\".","statusCode":400,"recommendedAction":"Provide listed parameters."})
```

Reply:
Updated